server/handlers/meshery_pattern_handler.go

```diff
@@ -315,6 +315,8 @@ func (h *Handler) handlePatternPOST(
 // ```?page={page-number}``` Default page number is 1
 //
 // ```?pagesize={pagesize}``` Default pagesize is 10
+//
+// ```?visibility={visibility}``` Default visibility is public
```

Comment:
run `make swagger-docs-build ` to update the docs

Reply:
Wouldn't it automatically run if the pr is merged?

Reply:
Can you please specify supported visibility levels as well?